app/controllers/all_casa_admins_controller.rb

```diff
@@ -11,7 +11,7 @@ def edit
   end
 
   def create
-    service = ::CreateAllCasaAdminService.new(params)
+    service = ::CreateAllCasaAdminService.new(params, current_user)
```

Comment:
Why does this need doing for docker?

Reply:
All of this was breaking the RSpec build in `main`. I can remove from this PR if you want.